### ISSUE TITLE
Ignore "Id" when followed by a period

### ISSUE THREAD

**Issue by benbalter** [retextjs/retext-contractions#6]
In legal writing, `Id.` is used as shorthand for "see previous citation". *See generally* http://tarlton.law.utexas.edu/bluebook-legal-citation/short-form.

If "Id" is followed immediately by a period, the project shouldn't suggest that `Id` become `I'd`. (I don't know when a sentence would end with "I would".

**Comment by wooorm** [comment]
Hey Ben!

The above commit fixes this, but while running `retext-contractions` I also found this line in [`2012-12-26-securing-the-status-quo.md`](https://github.com/benbalter/benbalter.github.com/blob/9ec654b952e7229b0db1a6190a8df6f542acb2bc/_posts/2012-12-26-securing-the-status-quo.md):

```markdown
[^22]: 5 U.S.C.A 552a.

[^23]: *Id* 552a(b), (d), (f)*.*

[^24]: 44 U.S.C.A. 3601 *et seq*.
```

Is the `Id` in `[^23]` the standard way of doing this? Or should it actually have a period?
Note: Your linked document suggests `Id. at 45.` if the page changes.
What do you think?

**Comment by benbalter** [comment]
> The above commit fixes this

Wow! Thanks. Hope the feedback's been helpful.

>  Or should it actually have a period

It should have a period. BlueBook (the standard or legal citation) is very clear about that (and that the period should be italicized — don't get me started).

**Comment by wooorm** [comment]
Good! Published as [`parse-latin@4.1.0`](https://github.com/wooorm/parse-latin/releases/tag/4.1.0), so an `npm install` should do the trick!

> It should have a period. BlueBook (the standard or legal citation) is very clear about that (and that the period should be italicized — don't get me started).

Alright, makes sense, then I’ll leave you to update `2012-12-26-securing-the-status-quo.md`!

👋
